EntAppProject/src/main/resources/templates/JBForgotPassword.html

```diff
@@ -2,42 +2,24 @@
 
 <html>
 
-<head>
-
-<title>Forgot Username/Password?</title>
-
-<link rel="stylesheet" href="JobBookCSS.css">
-
-</head>
-
-<body> 
-
-<h1><span>Job</span>Book</h1>
-
-<h2>Forgot Username/Password?</h2>
- 
-<form action="/JobBookWelcomeScreen">
-
-Email Linked to Account:     
-<input type="email" name="username">
-<br>
-<br>
- 
-New Password:                     
-<input type="password" name="password">
-
-<br>
-
-Confirm New Password:      
-<input type="password" name="password">
-<br>
-<br>
-                
-<input type="submit" value="Submit">
-</form>
-        
-<h4>Thanks!  Instruction on how to reset your password have been sent to your email.</h4>
-  
-</body>
-
-</html>
\ No newline at end of file
+    <head>

```

Comment:
I redid the HTML page restructuring from before but with the change of using the `page-title` class on span elements to specify that it should use the style included in `JobBookCSS.css`.